### PR TITLE
Microsoft MVP (Independent Publisher)

### PR DESCRIPTION
**Pull request by ahmadn82**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

image

### CODE REVIEW

File: independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3143 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft MVP",
+    "description": "The MVP Award Program is a community - and you\u2019ve always been there for each-other! A lot of you want to help us make this community better, including creating a better experience for submitting contributions. The MVP Connector will directly enable you to use your own ideas to create and enhance your own MVP experience.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json:5]: @ahmadn82 Please remove unicodes from the all the connector files as they may not be supported in all the Power Automate pages.

File: independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3143 @@
...
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GetContributionAreas",
+        "summary": "Get contribution areas",
+        "description": "Gets a list of Contribution areas grouped by Award Names",
+        "parameters": []
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json:73]: @ahmadn82 Can you please remove it, if not required ?

File: independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,3143 @@
...
+                      "description": "Contribution Technology Name",
+                      "title": "Name",
+                      "default": "",
+                      "x-ms-visibility": "internal"
+                    },
+                    "AwardName": {
+                      "type": "string",
+                      "description": "Award Name",
+                      "title": "Award Name",
+                      "default": "",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json:2943]: @ahmadn82 Can you please remove if do not have values for any property ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@natalie-pienkowska It is an IP connector submission for Microsoft Service. Please let me know to move forward for certification process.

**Comment by sriyen-msft** [comment]
@ahmadn82 Update - We are preparing your connector for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes between 7-10 business days. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@ahmadn82 Deployment freeze notice: Due to Thanksgiving week in the United States, we have temporarily halted the deployments of all the connectors till Tuesday, 30th November 2021. 

We will begin the deployments on Wednesday, 1st December 2021, and the production deployment of your connector in all regions will be expected to be completed by Thursday, 9th December 2021. Thanks

**Comment by sriyen-msft** [comment]
@ahmadn82 Apologies for the delay in response, we were seeing difficulties in setting up the OAuth app for your connector as the test account needs administrative or special permission. We are working with the concerned team to get the required permissions. I will keep you posted with the updates. Thanks.

**Comment by sriyen-msft** [comment]
@ahmadn82 Apologies for the delay in response, We are actively working on preparing and deploying your connector to publish environments. As you know that we are deploying the connector as a Standard connector, For that, we need to go thru a few guidelines and approvals from the concerned teams (ex. Power BI API team) which is a bit lengthy process. Please allow us some more time, we will keep you posted with the updates and deployment plan soon. Thank you so much for your patience.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by v-EgorKozhadei** [comment]
Closing this Pull Request upon partner's request because of the deprecated API.

**Comment by Amjed-Ayoub** [comment]
Thank you so much @ahmadn82 for the confirmation via email. We will close this PR.

**Pull request closed without merging**